frontend/invites/new_invite_page.html

```diff
@@ -46,34 +46,34 @@ <h3><b>{{ newInviteCtrl.invite.suggestion_institution_name }}</b></h3>
               <!-- USER INFORMATION -->
               <div layout="column" ng-if="newInviteCtrl.isUserInfoImcomplete()"
                style="margin-top:7px;">
-                <md-input-container ng-if="newInviteCtrl.checkUserName()" class="no-margin">
+                <md-input-container ng-if="newInviteCtrl.checkUserName()" class="zero-margin">
```

Comment:
add a class to the parent div, then use composition to change the margin of all md-input-container. e.g:
`.some-class md-input-container {
     margin: 0;
}`

Add that to the new_invite.css file

Reply:
Eu movi essa classe porque pretendo reutilizar ela na que estou fazendo. Se eu fizer isso não terá como.